### PR TITLE
Update Changelog for 2.0.0.1

### LINKED ISSUE #95: Create CHANGELOG and Readme for WSO2 Open Banking version 2.0.0
**Description:**
This PR serves $subject.

### PR DESCRIPTION
**Pull request by nalakagunathilaka**
## Purpose
Update changelog for v1.5.0.1. This fixes #95

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,41 +4,19 @@ All notable changes to Docker and Docker Compose resources for WSO2 Open Banking
...
-
-For detailed information on the tasks carried out during this release, please see the GitHub milestone
-[v1.5.0.2](https://github.com/wso2/docker-open-banking/milestone/10).
-
-## [v1.5.0.1] - 2020-05-22
-
-### Added
+- Open Banking UK and BERLIN specifications support	- Alpine and Ubuntu based Docker resources for WSO2 Open Banking API Manager `2.0.0`, WSO2 Open Banking Identity Server
+  as Key Manager version `2.0.0` and WSO2 Open Banking Business Intelligence Dashboard and Worker profiles `2.0.0`
+- Upgrade profiles to use Adopt OpenJDK `jdk8u272-b10`
```
Comment (DilanUA) [inline CHANGELOG.md:13]: make this "profile based dockerfiles to use so and so as the base image"

File: README.md [hunk 2]
```diff
@@ -6,14 +6,14 @@ This repository contains the following resources:
...
 
 Docker resources for WSO2 Open Banking API Manager, WSO2 Open Banking Key Manager and WSO2 Open Banking Business Intelligence help you build generic Docker images for deploying the corresponding product servers in containerized environments. Each Docker image includes the JDK, the relevant product distribution and a collection of utility libraries. Configurations, custom JDBC
 drivers, extensions and other deployable artifacts are designed to be provided via volume mounts to the containers spawned.
 
 Docker Compose resources have been created according to the most common WSO2 Open Banking deployment profiles available for allowing users to quickly evaluate product features along side their co-operate Open Banking requirements. The Compose resources make use of Docker images of WSO2 Open Banking API Manager, WSO2 Open Banking Key Manager and MySQL.
 
-**Change log** from previous `v1.5.0.2` release: [View Here](CHANGELOG.md)
+**Change log** from previous `v2.0.0.1` release: [View Here](CHANGELOG.md)
```
Comment (DilanUA) [inline README.md:16]: this has to be 1.5.0.2 right?

**Review by DilanUA — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (nalakagunathilaka) [reply README.md]: Changed. Thank you!

File: CHANGELOG.md (hunk 1 above)
Comment (nalakagunathilaka) [reply CHANGELOG.md]: Changed. Thank you!

**Review by DilanUA — APPROVED** [review]

**Pull request merged**